### DISCUSSION TITLE
Margin auto left and right not working on media query eg. md: lg:

### DISCUSSION THREAD

**Discussion by syahirudean** [tailwindlabs/tailwindcss#7199] [Help]
Hi, I can't seems to get `lg:mr-auto` and 'lg:ml-auto' to work.

`
<section class="border-2 border-gray-500">
    <div class="flex flex-col lg:flex-row">
        <div class="border-2 border-gray-500">
            <img class="w-full" src="https://picsum.photos/id/1/200/300" alt="" loading="lazy">
        </div>
        <div class="flex flex-col px-0 py-14 lg:p-0 bg-green-200">
            
            <div class="w-8/12 md:w-10/12 mx-auto lg:ml-auto mb-8 bg-red-400">
                <p class="mb-3">"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
                    incididunt
                    ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
                    nisi ut
                    aliquip ex ea commodo consequat.</p>

                <p class="mb-3">Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore
                    eu fugiat nulla pariatur.
                </p>
                <p>in voluptate velit esse cillum dolore
                    eu fugiat nulla pariatur.</p>
            </div>
            <div class="md:w-4/12 mx-auto lg:mr-auto">
                <button>READ MORE</button>
            </div>
        </div>
    </div>
</section>
`

**Comment by macbruker** [comment]
```html
<div class="md:w-4/12 mx-auto lg:mr-auto">
    <button>READ MORE</button>
</div>
```

What is the outcome you want here? 

`mx-auto` sets `margin-left` and `margin-right` to `auto`. Usually to center an element within its parent. `lg:mr-auto` also sets `margin-right` to `auto`, but only from the large breakpoint. Setting the same value twice, won't change the layout. If you want that part of the markup to behave differently at the large breakpoint, you need to provide a different value than what is already there. If you replace `lg:mr-auto` with either `lg:mr-0` or `lg:ml-0`, you should see a change at the large breakpoint, depending on how the rest of your layout is structured.
